README.md

```diff
@@ -185,4 +268,38 @@ kubectl exec -it -n <namespace> <pod-name> -- /bin/bash
 Get information about pod:
 ```
 kubectl describe pod -n <namespace> <pod-name>
-```
\ No newline at end of file
+```
+
+## Containerize your Client
+### Dockerfile Template
+```
+FROM ubuntu:22.04
+RUN apt-get update && apt-get install -y iputils-ping curl vim && \
+    rm -rf /var/lib/apt/lists/* && \
+    useradd -ms /bin/bash solana && \
+    adduser solana sudo
+
+USER solana
+COPY --chown=solana:solana ./target/release/<client-executable> /home/solana/
+COPY --chown=solana:solana ./client-accounts/ /home/solana/client-accounts/
```

Comment:
For the generic clients, this does seem slightly annoying -- since there's an environment variable for other things, how about putting the path to some `client-accounts.yml` file in an environment variable? Then the generic client can choose to use this or not

Reply:
not sure I fully understand, we still have to copy the client accounts file into the container when we build it. but say you need client-accounts.yml or whatever for your client, instead of doing `--client-accounts client-accounts.yml`, you would just do: `--client-accounts $CLIENT_ACCOUNTS`. and $CLIENT_ACCOUNTS would be set to `<path>/client-accounts.yml`?

Reply:
so we could have some function like this: 
```
    fn set_client_accounts_environment_variable(&self) -> EnvVar {
        k8s_helpers::create_environment_variable(
            "CLIENT_ACCOUNTS_PATH".to_string(),
            Some("/home/solana/client-accounts.yml".to_string()),
            None,
        )
    }
```

This would then enforce the client docker container, to copy whatever client accounts it needed from their host into the docker image on build. And they would have to copy them into: `/home/solana/client-accounts.yml`. would also force them to have a docker image set up with: `/home/solana/` as a valid path

I guess you could make that path configurable, and pass it in via the validator lab cli. but then it ends up being basically the same as passing the path in through the validator-lab, client sub command cli. 

Maybe I am not fully following.....

Reply:
I didn't express the idea clearly, sorry about that. I wanted to avoid the need to package the client accounts file, instead by "passing it in" to the client container somehow.

If the tool just sent a file called `client-accounts.yml` to the container, and the container knew to expect client accounts at a particular location, then the generic client setup could be a bit easier.

But since it's all docker containers, I'm realizing now that it might be goofy and less clear than just telling people to map client accounts into the container themselves.

Reply:
ok so you're on to something that can be done through `ConfigMap`s in kubernetes. So you basically can deploy the file in kubernetes on its own and can "mount" it into the pod on deployment. This makes it so you don't have to build the client file into the docker container. The problem is ConfigMaps have a max size limit of 1MB. so any client accounts file > 1MB could not be setup this way. 

There are a couple options here
1) `PersistentVolume`s, but i remember getting some push back from the monogon team for this. I forget why exactly 
2) Push a pod to docker hub that has a ton of client files on it. and put an RPC on it that will serve client files. And just deploy it with every cluster that needs a client. This is not really idea. but more of a hack. 
3) What we have now, build client file into the docker image.

Reply:
ok looking back at some of my messages with monogon team. It sounds like it may be possible to use persistent volumes. I think for now, maybe we can leave it as is (accounts baked into docker image). And then in the meantime, I will add in support for persistent volumes, so client accounts can be deployed separately from the client images.

Reply:
Ok great! I don't think it's a priority, btw, but good to know that it's possible in the future if needed